server/app/controllers/api/v1/api_users_controller.rb

```diff
@@ -1,10 +1,11 @@
 class Api::V1::ApiUsersController < ApplicationController
   before_action :set_api_user, only: [:show, :update, :destroy]
-
+  before_action :set_default_response_format
```

Comment:
nice idea! Do you think it makes sense to have it called by default in ApplicationController so that all controllers receive this by default? I don't see it applied  in the rest of the controllers in this review, so that might be something to consider.